commands/volumes/volume-option.go

```diff
@@ -58,6 +58,10 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		restutils.SendHTTPError(w, http.StatusBadRequest, fmt.Sprintf("invalid option specified: %s", err.Error()))
 		return
 	}
+	// Update the options into volinfo
+	for k, v := range req.Options {
+		volinfo.Options[k] = v
+	}
```

Comment:
There is a loop that does exactly this on line 106, before `txn.Do()` is called. Why doesn't it work there?